pages/events/index.tsx

```diff
@@ -0,0 +1,145 @@
+import { GetServerSideProps, NextPage } from 'next';
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+import moment from 'moment';
+import { Content, Main } from '@this/src/components/layout';
+import { getCalendarEvents, CalendarEventItem } from '@this/pages-api/calendar/events';
+
+type CalendarEventProps = {
+  calEvents: CalendarEventItem[];
+};
+
+const Description = ({ text }: { text: string }) => {
+  const [parsedText, setParsedText] = useState<string[]>([]);
+  useEffect(() => {
+    const el = document.createElement('div');
+    el.style.whiteSpace = 'pre';
+    el.innerHTML = text || '';
+    const newText = el.innerText.split('\n');
+
+    setParsedText(newText);
+  }, [text]);
+
+  return (
+    <div className='event-description'>
+      {parsedText.map((p, i) => (
+        <p key={`${p}-${i}`}>{p}</p>
```

Comment:
Why not just use `p` as the key?
```suggestion
        <p key={`${p}-${i}`}>{p}</p>
```

Reply:
in the off chance that there is a matching line it won't throw a warning (this happened while developing so I added the index)

Reply:
Wouldn't you want to know about that case, in case you put the same description in multiple times?